parks/parks-model.js

```diff
@@ -3,28 +3,59 @@ const mappers = require('../helpers/mappers')
 
 module.exports = {
     find,
-    findById,
+    findBy,
     add,
     findByPark,
+    addRating,
+    getParkRatings,
+    remove,
+    update
 };
 
 function find() {
-    return db('parks as p')
-        .join('ratings as r', 'p.id', 'r.park_id')
+    return db('parks')
+        .leftJoin('ratings', 'parks.id', 'ratings.park_id')
+        .select('parks.*', 'ratings.rating', 'ratings.comments', 'ratings.id as rating_id')
```

Comment:
How did you fix it? I thought you originally had it as a _leftJoin_.

Reply:
Not using that function anymore; created a findBy() function instead